src/test/java/org/mamute/infra/NormalizerBrutalTest.java

```diff
@@ -27,6 +32,9 @@ public void should_remove_question_marks_and_bangs() {
 		String text = "como faz para normalizar uma string?!";
 		String slug = NormalizerBrutal.toSlug(text);
 		assertEquals("como-faz-para-normalizar-uma-string", slug);
+
+		String slugEncoded = NormalizerBrutal.toSlug(text, true);
+		assertEquals("como-faz-para-normalizar-uma-string%3f%21", slugEncoded);
```

Comment:
If we start url encoding like this, we will lose the friendly urls we have for questions (example: http://demo.mamute.org/297-what-happens-if-i-use-strange-chars-on-the-title). 

Maybe we should keep the old normaliser and create a normaliser only for tags. What do you think, @MatejBalantic?